halo-ontology/.htaccess

```diff
@@ -0,0 +1,31 @@
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure .owl files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .owl
+
+# Rewrite engine setup
+RewriteEngine On
+
+# Base directory
+RewriteBase /halo-ontology/
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://github.com/navapatn/halo-ontology/blob/main/ontology/index-en.html [R=303,L]
+
+# Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^$ https://github.com/navapatn/halo-ontology/blob/main/ontology/ontology.owl [R=303,L]
+
+# Default response
+# ---------------------------
+# Rewrite rule to serve the RDF/XML content from the vocabulary URI by default
+RewriteRule ^$ https://raw.githubusercontent.com/navapatn/halo-ontology/main/ontology/ontology.owl [R=303,L]
+
+# Error document in case of a 406
+ErrorDocument 406 /halo-ontology/406.html
```

Comment:
/halo-ontology/406.html is not a local page. You should redirect to an html document, as you do above